server/src/main/scala/services/crunch/CrunchSystem.scala

```diff
@@ -186,11 +185,11 @@ object CrunchSystem {
       arrivalsStage, arrivalSplitsGraphStage,
       staffGraphStage, staffBatcher, deploymentGraphStage,
       forecastArrivalsDiffingStage, liveBaseArrivalsDiffingStage, liveArrivalsDiffingStage,
-      props.actors("forecast-base-arrivals").actorRef, props.actors("forecast-arrivals").actorRef, props.actors("live-base-arrivals").actorRef, props.actors("live-arrivals").actorRef,
+      props.actors("forecast-base-arrivals"), props.actors("forecast-arrivals"), props.actors("live-base-arrivals"), props.actors("live-arrivals"),
```

Comment:
while we are here, I think it would probably be better if we put all the parameters on their own line here, might be worth adding names in as well.

Reply:
:+1: